src/src/com/microsoft/aad/adal/UnexpectedServerResponseException.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.aad.adal;
+
+import java.io.IOException;
+
+/**
+ * Created by imakhal on 2/17/2016.
+ */
+public class UnexpectedServerResponseException extends IOException {
```

Comment:
My logic was that this is exception that sent from network calls and indicates type of error during network call. And I think this is actually failure of IO operation/network call.
But I don't mind to inherit from Exception.